src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -102,13 +102,10 @@ export class SortableTableComponent implements OnInit {
    * Sorts the table with an initial SortBy
    */
   initialSort(): void {
-    let indexOfColumnToSort: number =
+    const indexOfColumnToSort: number =
```

Comment:
Is there a reason for removing the left most column sort?

Reply:
The reason is the same as above. Using the left most column will lose the initial sorting of options.